nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/api/filter/TestCharacterFiltering.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * (C) Copyright 2017 Nuxeo SA (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
We'll clean this up later, but please try to keep the exact formatting of the Apache license, where this URL line is indented by 4 spaces.